mlflow/models/python_api.py

```diff
@@ -107,6 +107,7 @@ def predict(
     install_mlflow=False,
     pip_requirements_override=None,
     extra_envs=None,
```

Comment:
Didn't notice we have `extra_envs`. Does setting `UV_PRERELEASE` in `extra_envs` work?

Reply:
Yea that's basically what this PR does

Reply:
Sorry, I meant the the `extra_envs` parameter description.

Reply:
I believe it's fine. Might not be the most helpful way but we can improve it if requested. Most of users don't touch prereleases.